### PR TITLE
add nwbview

### PR DESCRIPTION
**Pull request by anilbey**
Hi folks :) 

A follow-up PR after the SfN conference. Addressing https://github.com/NeurodataWithoutBorders/nwb-overview/issues/123.

I built it locally it renders fine. The html newlines are added for it to look nice. Here is the screenshot below.

Screenshot from 2024-02-08 13-35-19

### CODE REVIEW

File: docs/source/tools/analysis_tools_home.rst [hunk 1]
```diff
@@ -87,6 +88,24 @@ Exploring NWB Files
...
+
+.. raw:: html
+
+    <br/>
+    <br/>
+    <br/>
+    <br/>
+    <br/>
+    <br/>
+    <br/>
```
Comment (oruebel) [inline docs/source/tools/analysis_tools_home.rst:107]:
```suggestion
```

**Comment by oruebel** [comment]
Looks like some links are broken. I'll make a separate PR to fix links and then this should be good to merge.

File: docs/source/tools/analysis_tools_home.rst [hunk 2]
```diff
@@ -87,6 +88,22 @@ Exploring NWB Files
...
+        :start-after: .. short_description_start
+        :end-before: .. short_description_end
+
+.. raw:: html
+
+    <br/>
+    <br/>
+    <br/>
+    <br/>
+    <br/>
```
Comment (oruebel) [inline docs/source/tools/analysis_tools_home.rst:105]:
```suggestion
```

File: docs/source/tools/analysis_tools_home.rst [hunk 3]
```diff
@@ -87,6 +88,20 @@ Exploring NWB Files
...
+
+.. include::  nwbview/nwbview.rst
+        :start-after: .. short_description_start
+        :end-before: .. short_description_end
+
+.. raw:: html
+
+    <br/>
+    <br/>
+    <br/>
```
Comment (oruebel) [inline docs/source/tools/analysis_tools_home.rst:103]:
```suggestion
```

**Comment by oruebel** [comment]
@stephprince could you take a look at the broken link issue? I tried to add the configuration for the linkchecker to ignore the link but for some reason it doesn't seem to work.

File: docs/source/conf.py [hunk 4]
```diff
@@ -72,5 +72,10 @@
     'titles_only': False
 }
 
+# these links cannot be checked in github actions
+linkcheck_ignore = [
+    " https://crates.io/crates/nwbview",
```
Comment (stephprince) [inline docs/source/conf.py:77]:
```suggestion
    "https://crates.io/crates/nwbview",
```

**Comment by oruebel** [comment]
Looks like the failing linkcheck is now a seperate issue with a timeout form biorxiv. It looks like https://www.biorxiv.org/ may be unavailable for some reason right now. I think we can probably just rerun the linkchecker again later and then merge.

**Comment by anilbey** [comment]
Great, thanks for your effort

**Review by oruebel — APPROVED** [review]

**Comment by oruebel** [comment]
Thanks @anilbey ! All tests passed now and I have merged the PR.

**Pull request merged**